test/signhash.ts

```diff
@@ -40,12 +40,89 @@ contract('SignHash', accounts => {
       const event: HashSigned = log.args;
 
       assert.equal(event.hash, HASH);
-      assert.equal(event.signer, accounts[0]);
+      assert.equal(event.signer, signer);
     });
 
     it('should reject empty hash', async () => {
+      const signer = accounts[0];
+      await assertThrowsInvalidOpcode(async () => {
+        await instance.sign('', { from: signer });
+      });
+    });
+  });
+
+  describe('#prove', () => {
+    it('should add proof', async () => {
+      const signer = accounts[0];
+      const method = 'http';
+      const value = 'example.com';
+      await instance.prove(method, value, { from: signer });
+
+      const proof = await instance.getProof(signer, method);
+      assert.equal(proof, value);
+    });
+
+    it('should add multiple proofs', async () => {
+      const signer = accounts[0];
+      const proofs = [
+        {
+          method: 'http',
+          value: 'example.com'
+        },
+        {
+          method: 'github',
+          value: 'example'
+        },
+        {
+          method: 'dns',
+          value: 'another.com'
+        }
+      ];
+
+      await Promise.all(proofs.map(x => instance.prove(x.method, x.value, { from: signer })));
+      await Promise.all(proofs.map(async x => assert.equal(await instance.getProof(signer, x.method), x.value)));
```

Comment:
- Aren't we autoformatting to 80 characters?
- `x` requires brain power to understand -> `proof`?

Reply:
Changed formatting to 80 columns instead of 120. Check if you like it: 242f40f13eee633f4ffc96c429fc9bd6fdd32c08